testWorkDir/Directory.Build.props

```diff
@@ -0,0 +1,3 @@
+<Project>
```

Comment:
This is something we'll have to do in all of our sample-like stuff right?

Reply:
We don't need to do it in samples in general. This is only required for dotnet-watch's tests because it is generating csproj files on the fly for its functional test.